.buildkite/pipeline-resource-definitions/kibana-gen-ai-daily.yml

```diff
@@ -0,0 +1,54 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: bk-kibana-gen-ai-daily
+  description: Runs the GenAI FTR tests daily
+  links:
+    - url: 'https://buildkite.com/elastic/kibana-gen-ai-daily'
+      title: Pipeline link
+spec:
+  type: buildkite-pipeline
+  owner: 'group:appex-ai-infra'
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / gen-ai-tests / daily
+      description: Runs the GenAI FTR tests daily
+    spec:
+      env:
+        SLACK_NOTIFICATIONS_CHANNEL: '#appex-ai-infra-alerts'
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
```

Comment:
Is there anything to do? I had a few failures during my experiments and we didn't receive any notifications in that channel.

Reply:
I think it should work, once this is materialized to a pipeline, and this setting is respected